metric_tank/aggmetrics.go

```diff
@@ -48,10 +48,13 @@ func NewAggMetrics(store Store, chunkSpan, numChunks, chunkMaxStale, metricMaxSt
 // periodically scan chunks and close any that have not received data in a while
 // TODO instrument occurences and duration of GC
 func (ms *AggMetrics) GC() {
-	ticker := time.Tick(ms.gcInterval)
-	for now := range ticker {
+	for {
+		unix := time.Duration(time.Now().UnixNano())
```

Comment:
everything in this logic (unix, period and diff) is actually in nanoseconds. (e.g. for period below , by multiplying a number in seconds by time.Second we're essentially converting to nanoseconds (see https://golang.org/src/time/time.go?s=13980:13999#L448). similar for time.Minute which is really a number in nanoseconds.

however you did help me uncover a different bug: ms.gcInterval was already a time.Duration, e.g. in nanoseconds so multiplying by time.Second made the number way too high.